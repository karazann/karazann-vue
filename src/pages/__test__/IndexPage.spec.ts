import { shallowMount, RouterLinkStub } from '@vue/test-utils'
import IndexPage from '../index.vue'

describe('IndexPage', () => {
    const build = () => {
        const options = {
            data() {
                return {}
            },
            stub: {
                RouterLinkStub
            }
        }

        const wrapper = shallowMount(IndexPage, options)

        return {
            wrapper
        }
    }

    it('renders the component', () => {
        // arrange
        const { wrapper } = build()

        // assert
        expect(wrapper.html()).toMatchSnapshot()
    })
})
