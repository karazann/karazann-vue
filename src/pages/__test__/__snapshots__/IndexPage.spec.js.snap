// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IndexPage renders the component 1`] = `"<h1 class=\\"test\\">Hello world</h1>"`;
